tests/phpunit/tests/multisite/cleanDirsizeCache.php

```diff
@@ -312,14 +312,14 @@ function test_5_5_transient_structure_compat() {
 
 		function _get_mock_5_5_dirsize_cache( $site_id ) {
 			return array(
-				"/uploads/sites/$site_id/2/2"              => array( 'size' => 22 ),
-				"/uploads/sites/$site_id/2/1"              => array( 'size' => 21 ),
-				"/uploads/sites/$site_id/2"                => array( 'size' => 2 ),
-				"/uploads/sites/$site_id/1/3"              => array( 'size' => 13 ),
-				"/uploads/sites/$site_id/1/2"              => array( 'size' => 12 ),
-				"/uploads/sites/$site_id/1/1"              => array( 'size' => 11 ),
-				"/uploads/sites/$site_id/1"                => array( 'size' => 1 ),
-				"/uploads/sites/$site_id/custom_directory" => array( 'size' => 42 ),
+				"wp-content/uploads/sites/$site_id/2/2"              => array( 'size' => 22 ),
```

Comment:
The 5.5 transient stored absolute paths, but that may be difficult to do here and still have the tests pass in different setups (w/r/t being installed in a subdirectory, custom content dir locations, etc).

I didn't get a chance to look into that earlier, though, just FYI

Reply:
Ta. I have added the full path back in 9fc77ecf5b4419b4337a50a9e422bb00d1b6f14d 

I'll see what I can do to get the function to play nice.

Reply:
I think I have the function playing nice in bd749dc5e96f6a9e07b1bc50183a45995dba7fbc